### PR TITLE
18.04 installer + PS3 to PS4 controller

### PR DESCRIPTION
**Pull request by erelson**
Follow-up to: https://github.com/fetchrobotics/fetch_robots/pull/45

Going to take a pass through related pages needing updating...

### CODE REVIEW

File: source/ps4.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  for PlayStation 4*).
+
+.. warning:: Third party controllers may not work. Only Sony PS4 controllers have been tested.
+
+Upgrade your Fetch robot's packages
+-----------------------------------
+Run the following to upgrade packages that are relevant::
+
+   sudo apt-get update
+   sudo apt-get install --only-upgrade ros-indigo-*
```
Comment (JeffWilson7) [inline source/ps4.rst]: Shouldn't these be `apt`?

Comment (moriarty) [reply source/ps4.rst]: apt vs apt-get doesn’t make a big difference but I do think it should be “-indigo-“

Comment (erelson) [reply source/ps4.rst]: s/indigo/melodic, yeah :+1:

File: source/release_notes.rst [hunk 2]
```diff
@@ -1,14 +1,76 @@
...
+
+ * fetch-melodic-config:  0.2-0 (replaces fetch-system-config)
+ * freight-melodic-config:  0.2-0 (replaces freight-system-config)
+
+Updated Fetch Packages:
+
+ * ros-melodic-fetch-bringup: 0.7.5-0 -> 0.7.5-0
+ * ros-melodic-freight-bringup: 0.7.5-0 -> 0.7.5-0
+
+ # No release of the below? TODO all versions below
```
Comment (moriarty) [inline source/release_notes.rst]: no release of these? or no updates?

Comment (moriarty) [reply source/release_notes.rst]: You've got two similar lists, I bumped the middle number with the melodic release, so most are x.8.x

Comment (erelson) [reply source/release_notes.rst]: As I mentioned offline, capturing loosely both releases this year. I haven't done the version numbers for the upcoming release as I wasn't sure which packages we'd actually release prior to talking to you today. (y)

**Pull request merged**